tools/static_word_checks.py

```diff
@@ -8,16 +8,37 @@ class StaticCheck:
 
     def __init__(self, replace: bool):
         self.full_file_checks = [
+            self.check_for_code_statement,
+            self.check_for_double_r,
+            self.check_for_double_u,
             # add more checks to the list as they are added
         ]
         self.line_checks = [
             self.check_for_noreturn,
-            self.check_for_double_r,
-            self.check_for_double_u,
             # add more checks to the list as they are added
         ]
         self.replace = replace
 
+    def check_for_code_statement(self, filename: str, content: str) -> bool:
+        if ".. code::" in content:
+            content = content.replace(".. code::", ".. code-block::")
+        if ".. code::" in content:
```

Comment:
Is it possible for this to be true given the update of content above?

Reply:
Sorry, this must be a duplicate